src/receive_imf.rs

```diff
@@ -1744,7 +1745,11 @@ async fn add_parts(
 
     let state = if !mime_parser.incoming {
         MessageState::OutDelivered
-    } else if seen || is_mdn || chat_id_blocked == Blocked::Yes || group_changes.silent
+    } else if seen
+        || is_mdn
+        || chat_id_blocked == Blocked::Yes
+        || group_changes.silent
+        || mime_parser.from.addr == STATISTICS_BOT_EMAIL
```

Comment:
If the chat is archived and muted by default, maybe there's no need in this check?

Reply:
Without this check, you get this annoying number-in-a-circle for the archived chats:

<img width="1024" height="473" alt="image" src="https://github.com/user-attachments/assets/df9d52db-331d-4a09-a5e9-5ac340d83f94" />

Reply:
But why make the logic more complicated when it doesn't matter?

Reply:
In fact this will make the logic less complicated, otherwise the fix in #6415 is harder to implement. TL;DR: `InSeen` messages should cause `MsgsNoticed` events, but `InNoticed` shouldn't. Probably we have some naming issue here (`MsgsSeen` would be better), but it's how it (almost) works currently. "Almost", because only `imap::Session::sync_seen_flags()` emits this event, but not `receive_imf`.